scripts/update-platform-version.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+const fs = require("fs");
+
+function main() {
+  const PLATFORM_VERSION = process.argv[2];
+  const FILENAME         = "rhmap-mbaas-config.json";
+  const mbaasConfig      = fs.realpathSync(FILENAME);
+  const mbaasConfigJson  = require(mbaasConfig);
+
+  const isInfoMap = o => o.metadata.name === "fh-mbaas-info" && o.kind === "ConfigMap";
+  mbaasConfigJson.objects.filter(isInfoMap)[0].data.version = PLATFORM_VERSION;
```

Comment:
@grdryn Is there a possibility that the first item in the filtered array will ever have data returned as undefined?

Reply:
@davidffrench it's possible if someone removes or changes the name of the ConfigMap. In that case, either the script will need to be updated to match on the new name, or removed if the ConfigMap is removed.  I'll add a case to fail if it's not matched.

wrt unit tests: there aren't any in this repo. The bash and node and go scripts in here are all quite simple, and are only here as a convenience to enable easier updating of the json templates. I don't believe unit tests would be worth the effort in this case.

Reply:
Perfect, thanks @grdryn

Reply:
> I'll add a case to fail if it's not matched.

Actually it will already fail if `data` doesn't exist, so nothing to do there